lib/data/constants/app_assets.dart

```diff
@@ -0,0 +1,8 @@
+//shared_preferences key
+const String formDetailsKey = 'bank_detail_key', userKey = 'user_key';
```

Comment:
app assets is used for saving assets name ie icons images lotties etc

Reply:
Dont put these keys and stuff here. Put them in some main file where keys need to be declared and use it from there